src/app/core/models/github-user.model.ts

```diff
@@ -52,6 +52,22 @@ export class GithubUser implements RawGithubUser, Group {
     Object.assign(this, rawData);
   }
 
+  static fromUsername(username: string) {
```

Comment:
Might be useful to add a comment to explain why this method exists so other devs would the purpose of this function.